package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "@onna/tracking",
-    "version": "0.0.0",
+    "name": "onna-tracking",
```

Comment:
Nathan said yesterday we haven't bought the "@onna" domain.
But we can use "@atlasense" for public packages if we want.

Reply:
But will be a bit confusing that the repository is @onna/onna-tracking and the package is @atlasense/tracking or @atlasense/onna-tracking